traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -155,8 +156,33 @@ WHERE server.status NOT IN (
      AND server.cdn_id  =  (
                              SELECT deliveryservice.cdn_id
                              FROM deliveryservice
-                             WHERE deliveryservice.%s=$1
+                             WHERE deliveryservice.id=$1
+                           );
+`
+
+const queueRevalQuery = `
+UPDATE public.server
+SET revalidate_update_time = now()
+WHERE server.status NOT IN (
+                             SELECT status.id
+                             FROM status
+                             WHERE name IN ('OFFLINE', 'PRE_PROD')
+                           )
+     AND server.profile IN (
+                             SELECT profile_parameter.profile
+                             FROM profile_parameter
+                             WHERE profile_parameter.parameter IN (
+                                                                    SELECT parameter.id
+                                                                    FROM parameter
+                                                                    WHERE parameter.name='location'
+                                                                     AND parameter.config_file='regex_revalidate.config'
+                                                                  )
                            )
+     AND server.cdn_id  =  (
+                             SELECT deliveryservice.cdn_id
+                             FROM deliveryservice
+                             WHERE deliveryservice.id=$1
+                           );
```

Comment:
I can and will fix this.

Just to explain my reasoning: Code duplication isn't inherently bad. Specifically if it has to do with two different logical paradigms or domains. In this case, config updates are not equivalent reval updates and they could change over time in their expected behavior and functionality. In general, trying to keep a function, or query, or handler, or whatever compliant for mulitple scenarios can increase complexity in an effort to reduce duplicated lines. In some cases it can lead towards [significantly] more code, as well as increased cyclomatic complexity, as all scenarios have to be accounted for. I tend to lean towards Single Responsibility Principle over duplicated code.

Also I removed the string injection from the sql query out of habit as a way to generally avoid SQL injection, and depending on the SQL library allows for better query caching. Granted that's not really a problem here, but when I see it my reaction is to remove it.